README.md

```diff
@@ -103,32 +108,52 @@ Note: These instructions have primarily been tested for Mac/Linux/WSL environmen
 
 3. Create an IoT TwinMaker workspace.
 
-   Different Digital Twin applications use different resources. Run the following command to create a role for our workspace that has the necessary permissions for this sample application. Note that you will use the role name in the output in the next step.
+   a. Create a TwinMaker execution role
```

Comment:
"IoT TwinMaker" (may be other places)

Reply:
Updated everywhere in the README and grafana_local_docker_setup.md